packages/flutter/lib/src/material/tooltip.dart

```diff
@@ -851,7 +845,7 @@ class TooltipState extends State<Tooltip> with SingleTickerProviderStateMixin {
       onEnter: _handleMouseEnter,
       onExit: _handleMouseExit,
       decoration: widget.decoration ?? tooltipTheme.decoration ?? defaultDecoration,
-      textStyle: widget.textStyle ?? tooltipTheme.textStyle ?? defaultTextStyle,
+      textStyle: defaultTextStyle.merge(tooltipTheme.textStyle).merge(widget.textStyle),
```

Comment:
could you try `(tooltipTheme.textStyle ?? defaultTextStyle).merge(widget.textStyle)` and see if it still affects the test images?

Reply:
Still no luck. It might be best to put the text style merging on hold for now and proceed with just the following change:

```diff
child: DefaultTextStyle(
- style: Theme.of(context).textTheme.bodyMedium!,
+ style: textStyle,
+ textAlign: textAlign,
```

This would resolve the original issue, which was that the default text style didn't reach all the way down to text widgets hidden beneath widget spans.

We can open a separate issue for the breaking change related to text style merging, and later decide whether to proceed with it. What do you think?